src/cipher/chacha20/go-chacha20/chacha/chacha20_ref.go

```diff
@@ -1,5 +1,3 @@
-// This was originally "+build !cgo" but the cgo file is always ignored in build now
```

Comment:
Leave this comment
Figure out a different way to suppress the error

Can try adding `//` as the first line of the file